rubato/struct/scene.py

```diff
@@ -28,9 +28,9 @@ class Scene:
 
     def __init__(
         self,
-        name: str = "default",
+        background_color: Color = Color(255, 255, 255),
         border_color: Color = Color(),
-        background_color: Color = Color(255, 255, 255)
+        name: str = "default",
```

Comment:
Is there a reason name isn't first anymore. `intro_scene = Scene("intro")` I believe is important.